useful-scripts/get_data_block_responses/get_data_block_responses.py

```diff
@@ -172,6 +172,7 @@ def write_this_block(status_code, response_time, response_dict):
 
     try:
         json_response = json.loads(resp.text)
+
     except Exception as e:
         print_to_console(f'Failed to convert response text to json with block {block_id} subject {subject_id}, {e}')
```

Comment:
In this print_to_console_I think it would be handy to include `resp.status_code`